src/components/MultipleChoiceQuestion.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import Immutable from 'immutable';
+
+import Dropdown from 'components/Dropdown';
+
+export default class MultipleChoiceQuestion extends PureComponent {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    question: PropTypes.instanceOf(Immutable.Map).isRequired,
+    nextQuestionOptions: PropTypes.array.isRequired,
+  };
+
+  render() {
+    const { id, question, nextQuestionOptions } = this.props;
+    const defaultNextQuestionId = question.get('default_next_question_id');
+    return (
+      <div>
+        <div>{question.get('text')}</div>
+        <div style={styles.options}>
+          { question.get('options') && question.get('options').map((option, index) => (
+            <div key={index} style={styles.option}>
+              <div
+                style={{
+                  ...styles.key,
+                  ...styles.item,
+                }}
+              >
+                {option.get('key')}
+              </div>
+              <div
+                style={{
+                  ...styles.optionText,
+                  ...styles.item,
+                }}
+              >
+                {option.get('text')}
+              </div>
+              <div
+                style={{
+                  ...styles.nextQuestionDropdown,
+                  ...styles.item,
+                }}
+              >
+                { /* TODO(ayushbhagat): Handle the case where default_next_question_id === -1 and
+                option.next_question_id === -1 (i.e. the conditional question is the last
+                question). */ }
+                <Dropdown
+                  selectedValue={defaultNextQuestionId === -1
+                    ? `${option.get('next_question_id')}`
+                    : `${defaultNextQuestionId}`}
+                  options={{
+                    ...nextQuestionOptions.map(questionId => id !== questionId && ({
+                      value: `${questionId}`,
+                      words: `Continue to Question ${questionId + 1}`,
+                    })).reduce((acc, cur) => ({ ...acc, [cur.value]: cur.words }), {}),
+                  }}
+                  onChange={newValue => console.log(newValue)}
+                  width={231}
```

Comment:
231 ? Seems random, are these constants ? Should we define them in `constants/` instead ?